Post/templates/Post/postsList.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+	<title>list of posts</title>
```

Comment:
maybe "Post List" will be better phrasing and looking better?

Reply:
I changed it.